api-tests/src/config/config.ts

```diff
@@ -2,13 +2,12 @@ import dotenv from "dotenv";
 
 const CORE_ENV = process.env.CORE_ENV;
 
-if (!CORE_ENV) {
-  dotenv.config();
-} else {
+if (CORE_ENV) {
   dotenv.config({
     path: `.env.${CORE_ENV}`,
   });
 }
+dotenv.config();
```

Comment:
Config is 'first wins' (by default) - this means that we'll have the precedence:
1. Environment variables
2. `.env.<name>`
3. `.env`